<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_6</class>
 <widget class="QDialog" name="fct_comp_6">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aperçu des représentations</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="header">
     <item>
      <widget class="QLabel" name="label_1_fct_comp_6">
       <property name="text">
        <string>Choisissez un spectacle :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_fct_comp_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table_fct_comp_6">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>dateRepr</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nbPlacesReserv</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_error_fct_comp_6">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_fct_comp_6</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>fct_comp_6</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
 </slots>
</ui>
